src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+    <p className="Person__age">
+      {person.age && (
+        `I am ${person.age}`
+      )}
+    </p>
+
+    {!person.isMarried ? (
```

Comment:
Handle the truth case first. It's easier to read and understand
```suggestion
    {person.isMarried ? (
```